KOGITO-6868 Added documentation for Source Files Addon (#4199)
kiegroup/kie-docs: doc-content/kogito-docs/src/main/asciidoc/release-notes/ReleaseNotesKogito.1.21.0.Final/ReleaseNotesKogito.1.21.0.Final.adoc

[role="_abstract"]
The following sections describe some of the new features or enhancements in {PRODUCT} 1.21.

== {PRODUCT} runtimes

=== Added new Source Files Add-on

> A new Add-on named <<source-files-add-on, Source Files>> that allows you to download the source files (`.bpmn`, `.bpmn2`, `.json`, and `.yaml`) was added.

Reviewer: ```suggestion
A new <<source-files-add-on, Source Files>> add-on that allows you to download the source files (`.bpmn`, `.bpmn2`, `.json`, and `.yaml`) are now added.
```
